en/backup-and-restore-using-helm-charts.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Backup and Restore Using Helm Charts
 summary: Learn how to back up and restore data of the TiDB cluster in Kubernetes using Helm charts.
-aliases: ['/docs/tidb-in-kubernetes/stable/backup-and-restore-using-helm-charts/','/docs/tidb-in-kubernetes/v1.1/backup-and-restore-using-helm-charts/','/docs/dev/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.1/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/how-to/maintain/tidb-in-kubernetes/backup-and-restore/','/docs-cn/dev/tidb-in-kubernetes/maintain/backup-and-restore/restore/']
+aliases: ['/docs/tidb-in-kubernetes/stable/backup-and-restore-using-helm-charts/','/docs/tidb-in-kubernetes/v1.1/backup-and-restore-using-helm-charts/','/docs/dev/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.1/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/how-to/maintain/tidb-in-kubernetes/backup-and-restore/','/docs-cn/dev/tidb-in-kubernetes/maintain/backup-and-restore/restore/','/docs/dev/tidb-in-kubernetes/maitain/backup-and-restore/']
```

Comment:
,'/docs-cn/dev/tidb-in-kubernetes/maintain/backup-and-restore/restore/','/docs/dev/tidb-in-kubernetes/maitain/backup-and-restore/'] -> ,'/docs/dev/tidb-in-kubernetes/maitain/backup-and-restore/']

Reply:
@TomShawn

Reply:
Thanks for pointing out. I'll create a PR to fix today.

Reply:
Fixed in https://github.com/pingcap/docs-tidb-operator/pull/821/files#diff-3d0c885b197d038e95373e699a61202e8efaeb924d6b5b498210ec6b99172364R4.